lib/internal/policy/manifest.js

```diff
@@ -475,6 +475,35 @@ class Manifest {
     return this.#createResolver(() => null);
   }
 
+  mightAllow(url, onreact) {
+    const href = `${url}`;
+    debug('Checking for entry of %s', href);
+    if (this.#resourceIntegrities.has(href)) {
+      return true;
+    }
+    let scope = this.#findScopeHREF(href);
+    while (scope !== null) {
+      if (this.#scopeIntegrities.has(scope)) {
+        const entry = this.#scopeIntegrities.get(scope);
+        if (entry === true) {
+          return true;
+        } else if (entry === kCascade) {
+        } else {
+          break;
+        }
+      }
+      const nextScope = this.#findScopeHREF(new URL('..', scope));
+      if (!nextScope || nextScope === scope) {
+        break;
+      }
+      scope = nextScope;
+    }
+    if (onreact) {
+      this.#reaction(onreact());
+    }
+    return false;
+  }
```

Comment:
Is this being covered by the tests?

Reply:
permutation 16 (of many) of test/pummel/test-policy-integrity.js actually is where this came up and made me use a callback instead of the `throw` in the older version of this PR.

Reply:
That sounds like a "yes". If so, could somebody resolve this thread so I know there's no action to take / exclude it from my to-do list 🙏